src/board-explorer-app/board-explorer-app.html

```diff
@@ -1554,12 +1562,37 @@
       if (this.selectedOnceLoaded) {
         var category = this.$.boardViewer.findCategory(this.selectedOnceLoaded[0]);
         if (category) {
-          this.category = category;
-          this.selected = category.list;
+          for (var i = 0; i < this.categories.length; i++) {
+            if (this.categories[i].refdes == this.selectedOnceLoaded[0].toLowerCase()) {
+              this.$.searchTabs.select('categoriesPage');
+              this.categories[i].selected = true;
+              console.log(this.categories[i]);
```

Comment:
@juliocri This may need to use the [Object and Array modifiers](https://www.polymer-project.org/1.0/docs/devguide/model-data). For example:

```javascript
             this.set("categories." + i + ".selected", true);
```